config/migrations/2025/20250121103936-extended-sync-op/20250121103950-flush-all-admin-units-before-sync-with-op.sparql

```diff
@@ -0,0 +1,602 @@
+PREFIX besluit: <http://data.vlaanderen.be/ns/besluit#>
+PREFIX ere: <http://data.lblod.info/vocabularies/erediensten/>
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+
+#######################################################################
+# START
+# First remove the extra mappings.
+# This based on the conversion from the export.json of the op-producer.
+#######################################################################
+DELETE {
+  GRAPH <http://mu.semte.ch/graphs/public> {
```

Comment:
Data seems to be mostly in `http://mu.semte.ch/graphs/access-for-role/PubliekeBesluitendatabank-BesluitendatabankLezer`, but some also in the public graph. The flushing query probably needs to be extended a bit

Reply:
Good to know, should we just add the `http://mu.semte.ch/graphs/access-for-role/PubliekeBesluitendatabank-BesluitendatabankLezer` graph to the queries like:

```
  VALUES ?g {
    <http://mu.semte.ch/graphs/public>
    <http://mu.semte.ch/graphs/access-for-role/PubliekeBesluitendatabank-BesluitendatabankLezer>
  }
  ```

Reply:
or just in `<http://mu.semte.ch/graphs/access-for-role/PubliekeBesluitendatabank-BesluitendatabankLezer>`, see the latest comment on the PR, depending on what can be found in the public graph! Because when you'll consume, consumed data will only go into `<http://mu.semte.ch/graphs/access-for-role/PubliekeBesluitendatabank-BesluitendatabankLezer>` if I understand correctly

Reply:
Yes you're completely right!